ui/src/auth/auth0OAuth.tsx

```diff
@@ -37,26 +37,28 @@ function Auth0ProviderWithClient() {
     user,
     isLoading,
     isAuthenticated,
+    getAccessTokenSilently,
     getIdTokenClaims,
     loginWithRedirect,
     logout,
   } = useAuth0();
 
-  // This is needed for user state to get updated.
+  // Needed for user state to get updated: calls auth0 if token is expired
   useEffect(() => {
     if (isAuthenticated) {
-      getIdTokenClaims();
+      getAccessTokenSilently();
```

Comment:
Just curious about the function change -- this token is still an ID token right?

Reply:
Yes, we still get the id token when we get the auth token at line 78. 

the reason for this function change - 
from the docs - https://auth0.github.io/auth0-react/interfaces/Auth0ContextInterface.html#getAccessTokenSilently

getAccessTokenSilently will automatically invoke login flow is there is no valid token (does not check expiry). 
Updating the comment